### ISSUE TITLE
WCAG2Mobile: Draft feedback - 2.2.2 Pause, Stop, Hide

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#662]
(Mary Jo) 

- Again "view/views" used. This change from WCAG2ICT isn't really necessary. 

Chris L : Agreed.

**Comment by pday1** [comment]
Closed - insufficient time or resource to review each WCAG2Mobile clause
